utils/apollo/schema/typedefs/index.js

```diff
@@ -42,6 +42,21 @@ export const typeDefs = gql`
     updatedAt: String
   }
 
+  input SpaceInput {
+    name: String
+    description: String
+    spaceId: ID
+    isActive: Boolean
+    music: String
+    userId: ID
+  }
+
+  type Mutation {
+    createSpace(input: SpaceInput): Space
+    # updateSpace(input: SpaceInput): Space
+    addUserToSpace(input: SpaceInput): Space
+    removeUserFromSpace(input: SpaceInput): Space
+  }
```

Comment:
Actually, I just learned that the better way of creating a typedefs file is to name the file `schema.graphql`, that way you get syntax highlighting. Please reference https://github.com/LemonFace0309/graphql-reference-v2/tree/main/graphql-basics for more details on what I mean. It'll only take 3 minutes.

Reply:
Ah, we have to add a loader to get the syntax highlighting in our webpack config file (Which we don't have yet). Maybe this should be for another PR
https://www.apollographql.com/docs/react/integrations/webpack/